engines/router/missionctl/fiberapi/routing_strategy.go

```diff
@@ -58,55 +59,59 @@ func (r *DefaultTuringRoutingStrategy) SelectRoute(
 	options := runner.GetTreatmentOptions{
 		TuringRequestID: turingReqID,
 	}
-	expPlan, expErr := r.experimentEngine.
-		GetTreatmentForRequest(req.Header(), req.Payload(), options)
 
-	// Create experiment response object
-	experimentResponse := experiment.NewResponse(expPlan, expErr)
-	// Copy experiment response to the result channel in the context
-	expResultCh, expChErr := experiment.GetExperimentResponseChannel(ctx)
-	if expChErr == nil {
-		expResultCh <- experimentResponse
-		close(expResultCh)
-	}
-
-	// If error, log it and return the fallback(s)
-	if expErr != nil {
-		log.WithContext(ctx).Errorf(expErr.Error())
-		return nil, fallbacks, nil
-	}
+	if req.Protocol() != protocol.GRPC {
```

Comment:
nit: Little advice, nominal case should be indented as minimal as possible. i.e. instead of wrapping the nominal case (protocol.HTTP), we can exit early and leave the rest of the code untouched.

```go
if req.Protocol() == protocol.GRPC {
    return return nil, fallbacks, nil
}

```